inductiva/inductiva
# Better table formatting

Adds and applies more formatters to the tables to get better visual results.
This pull request addressed https://github.com/inductiva/tasks/issues/78
### Main changes

The main change is that now I allow the formatters to be a mapping from columns to List[functions] so that more than one formatter can be applied to the columns.
I also added a new argument that allows to apply formatters to the headers.

### New functionalities

We now have a new environment variable that allows to control whether or not we add text-emphasisers ("red", "green", "bold") to the tables.

### QA

Just use the inductiva client for tasks, resources and storage. You should see something like this:
<img width="1159" alt="Screenshot 2024-02-01 at 11 14 31" src="https://github.com/inductiva/inductiva/assets/33664950/5291fad1-f00d-4645-8fe2-242fbc1da577">
<img width="1431" alt="Screenshot 2024-02-01 at 11 14 51" src="https://github.com/inductiva/inductiva/assets/33664950/79755c23-c971-4c3e-bffe-961ba2e964fd">
<img width="516" alt="Screenshot 2024-02-01 at 12 06 50" src="https://github.com/inductiva/inductiva/assets/33664950/f2746e61-1e8f-4e3a-8582-c35072395b31">

Finally you should also `pip` these outputs to a file, e.g, `inductiva tasks list > some_file.txt`. This should automatically remove all bolds and colors. Test by running `cat some_file.txt`
